common/index.ftd

```diff
@@ -7,6 +7,8 @@
 
 -- integer logo-width: 200
 
+-- string site-name: SET 2 DS
```

Comment:
@MeenuKumari28 site-name logic should be same as doc-site. Check how its implemented in doc-site and header component.